docs/calico-with-docker/docker-network-plugin/CalicoAsService.md

```diff
@@ -0,0 +1,86 @@
+<!--- master only -->
+> ![warning](../../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.14.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Running Calico Node Containers as a Service
+
+The `calico/node` and `calico/node-libnetwork` containers can be run as individual
+system processes or services.  This document explains how to run each node image on its own.
+
+## Running the Calico Node Containers as Systemd Services
+Included here are two sample systemd service files to start each Calico node image as a service.
+
+`calico-node.service`
+
+```
+[Unit]
+Description=calico-node
+After=docker.service
+Requires=docker.service
+
+[Service]
+ExecStartPre=@-/usr/bin/docker rm -f calico-node
+ExecStart=/usr/bin/calicoctl node
+
+[Install]
+WantedBy=multi-user.target
+```
+
+`calico-libnetwork.service`
+
+```
+[Unit]
+Description=calico-libnetwork
+After=docker.service
+Requires=docker.service
+
+[Service]
+ExecStartPre=@-/usr/bin/docker rm -f calico-libnetwork
+ExecStart=/usr/bin/docker run -d --privileged --net=host -v /run/docker/plugins:/run/docker/plugins --name=calico-libnetwork -e ETCD_AUTHORITY=localhost:2379 calico/node-libnetwork:latest
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Be sure to update the ETCD_AUTHORITY environment variable to point at the
+correct location where your Etcd datastore is running.
+
+Also, depending on how you've installed Docker, the name of the Docker under the `[Unit]` section may be different (such as being docker-engine.service).  Be sure to check this before starting the service.
+
+### Starting `calico/node-libnetwork` on its own
+
```

Comment:
Maybe start with an "If you are using Calico as a Docker network plugin..." as I mentioned above.